src/test/java/com/easypost/EventTest.java

```diff
@@ -32,11 +32,7 @@ public void testEventWithNullPreviousAttributes() {
         eventWrapper.put("mode", "production");
         eventWrapper.put("previous_attributes", null);
         eventWrapper.put("pending_urls", new ArrayList<String>(
-                Arrays.asList(
-                        "hook_4ed7ff31c6974ff180e2463bb2912345",
-                        "hook_b43bc6d4eb3848bfaaee8e131f712345")
-                )
-        );
+                Arrays.asList("hook_4ed7ff31c6974ff180e2463bb2912345", "hook_b43bc6d4eb3848bfaaee8e131f712345")));
```

Comment:
Same as above, I'd rather have comma separated lists be one item per line. Makes managing and scanning through them much easier.

Reply:
We can do this manually, but the .editorconfig settings is going to keep reverting.
Here, `asList` is actually a function that can take an infinite number of arguments. But since it's a function, it adheres to the rules about how functions with parameters are formatted. So the rule to change would impact all functions.
Under the current rules, like any function, if the number of parameters exceeds the line length limit, it will kick it down to a second line.

Reply:
I see, this is a function (I hadn't realized) and not a free-form list. This is probably fine as is. Thanks for the info!